services/order.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, MovieSession, Ticket
+
+User = get_user_model()
+
+
+def create_order(
+    tickets: list[dict[str, int]],
+    username: str,
+    date: Optional[datetime] = None
+) -> None:
+    user = User.objects.get(username=username)
```

Comment:
Consider using `get_object_or_404` or handling the `DoesNotExist` exception when retrieving the user by username. This will help manage cases where the username does not exist in the database.